scripts/test_rpc.py

```
@@ -0,0 +1,24 @@
+
+    time.sleep(1)
+
+    print('Response:')
+    timeout = time.time() + 3
+    while time.time() < timeout:
+        try:
+            line = ser.readline()
+            if line:
+                print(line.decode().strip())
+                break
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)